src/index.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-unresolved
+import '@fortawesome/fontawesome-free/js/fontawesome.js';
+// eslint-disable-next-line import/no-unresolved
+import '@fortawesome/fontawesome-free/js/regular.js';
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.